src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,120 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOADING_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private int size = 0;
+    private int capacity = 16;
+
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
+
+    public int getIndex(K key) {
+        return Math.abs(Objects.hash(key) % capacity);
```

Comment:
```suggestion
        return key == null ? 0 : Math.abs(Objects.hash(key) % capacity);
```
так тобі не потрібен буде if (key == null) { в путі